stdlib/public/core/MemoryLayout.swift

```diff
@@ -40,7 +40,12 @@
 ///             byteCount: count * MemoryLayout<Point>.stride,
 ///             alignment: MemoryLayout<Point>.alignment)
 @frozen // namespace
-public enum MemoryLayout<T> {
+public enum MemoryLayout<T: ~Copyable> {}
```

Comment:
This now makes `MemoryLayout` a noncopyable type, which is fine because you can never have an instance of it, but this does change some of the value witness table stuff regarding it. Perhaps it's fine, but if you wanted to be safe I would make this `Copyable`.

Reply:
Ah, good catch!